06-auto-scaling/README.md

```diff
@@ -389,16 +441,22 @@ update your stack again.*
 _Is your resulting configuration more or less complicated than the one that
 uses a simple policy?_
 
+> Less complicated
+
 Consume CPU the way you did in lab 1, then stop.
 
 ##### Question: Scale-Out Delay
 
 _How long do you have to let it run before you see the group scale out?_
 
+> About 2 minutes
```

Comment:
The default for the basic monitoring interval is 5 minutes. Detailed monitoring would be 1 minute.